### DISCUSSION TITLE
JWSError JWSInvalidSignature on new Database functionality

### DISCUSSION THREAD

**Discussion by brywhi** [ToolJet/ToolJet#5391] [Q&A]
Hello,
I'm using ToolJet in DigitalOcean following these instructions:
https://docs.tooljet.com/docs/setup/digitalocean/

I've enabled the new Tooljet Database functionality, but when I try to access it, I get this error:
![image](https://user-images.githubusercontent.com/42787119/213765700-b15a78af-54bd-40d4-bcf3-53b36f0b3ae5.png)

I'm not seeing anything useful in the DO runtime logs. Where should I look to debug this?

**Comment by adishM98** [comment]
Hello, 

Could you please check if you have entered the variable PGRST_JWT_SECRET.

**Comment by brywhi** [comment]
Yes, i have on both the tooljet app env and postgrest too.

**Comment by adishM98** [comment]
May i know how many characters is the JWT secret ? It should be of 32 character. You can generate it in you CLI using the command `openssl rand -hex 32`.
